layouts/shortcodes/upcoming-releases.html

```diff
@@ -0,0 +1,21 @@
+<div>
+  <table class="table table-striped col-md-9">
+    <thead class="thead">
+      <tr>
+        <th>{{ T "monthly_patch_release" }}</th>
+        <th>{{ T "release_cherry_pick_deadline" }}</th>
+        <th>{{ T "release_target_date" }}</th>
+      </tr>
+    </thead>
+    <tbody>
+      {{ range $patchReleaseInfo :=
+      .Site.Data.releases.schedule.upcoming_releases }}
+      <tr>
+        <td>{{ $patchReleaseInfo.release }}</td>
+        <td>{{ $patchReleaseInfo.cherryPickDeadline }}</td>
+        <td>{{ $patchReleaseInfo.targetDate }}</td>
```

Comment:
Looking at the current contents of the table in the Chinese file, it appears that there may be a need for additional logic to assist in localizing these strings.

Reply:
We should allow all three dates to be customized the way that a given localization prefers.

Reply:
Hm, how can this work?

Reply:
I think we do not even need the `patchReleaseInfo.release` field at all, we can just format the target date into the month + year.